sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -695,6 +695,12 @@ func withDefaultMounts(containerID string, persistentDir string) oci.SpecOpts {
 			Source:      "cgroup",
 			Options:     []string{"ro", "nosuid", "noexec", "nodev"},
 		},
+		// Bottlerocket release information for the container
+		{
+			Options:     []string{"bind", "ro"},
+			Destination: fmt.Sprintf("/etc/bottlerocket-release"),
```

Comment:
nit: I do like the simplicity of just having `/etc` instead of `/.bottlerocket/bottlerocket-release`, but I still think we should have the prefix just to be consistent with the container's persistent storage mount: `/.bottlerocket/<host-containers>/<bootstrap-containers>/`

Reply:
I went back and forth on this as well. As a general rule, we try to be hands-off with the container's root filesystem, but we've bent that rule to put things in the "expected" places - `apiclient` in `/usr/local/bin`, and the API socket in `/run`.

Having the release file in `/etc` seems like a reasonable exception by that standard. And if we're touching `/etc` in any case, making it a symlink to `/.bottlerocket/<blah>/etc` just seems overly complicated.